opendbc/dbc/hyundai_canfd.dbc

```diff
@@ -658,6 +662,139 @@ BO_ 1264 LOCAL_TIME: 8 XXX
  SG_ MINUTES : 21|6@0+ (1,0) [0|63] "" XXX
  SG_ SECONDS : 31|8@0+ (1,0) [0|59] "" XXX
 
+BO_ 353 MSG_161: 32 CCNC
```

Comment:
Same

Reply:
May be a good idea  to start thinking about separating the DBCs, and import hyundai_canfd if it's a superset for architecture clarity. For now you can give these a good suffix/prefix to differentiate them

Reply:
> Same

Done